controllers/kafkaacl_controller.go

```diff
@@ -71,8 +72,11 @@ func (h KafkaACLHandler) createOrUpdate(ctx context.Context, avn *aiven.Client,
 		return err
 	}
 
-	// New created ACL id set
-	acl.Status.ID = r.ID
+	// New-created ACL id set
+	acl.Status.ID, err = h.getID(ctx, avnGen, acl)
+	if err != nil {
```

Comment:
`ServiceKafkaAclAddIn` returns a list of ACLs. The old client also tries to "guess" ACL's ID, like `getID` here.
https://github.com/aiven/aiven-go-client/blob/main/kafka_acl.go#L48